tests/ml/serving/test_predictor.py

```diff
@@ -1,16 +1,95 @@
+import json
+import os
 import unittest
 
+import numpy as np
+
+from ml.serving import model_io
 from ml.serving.predictor import BanditPredictor
+from sklearn.utils import shuffle
 from tests.fixtures import Params, Datasets
+from workflows import train_bandit
+
+
+TMP_MODEL_PATH = os.path.dirname(os.path.abspath(__file__)) + "/tmp.pkl"
 
 
 class TestPredictor(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        pass
+        cls.tol = 0.001
```

Comment:
i assume tol = tolerance and is pretty standard abbrev?

Reply:
yup!